README.md

```diff
@@ -132,6 +132,17 @@ To enable Google Anaytics, add the following lines to your Jekyll site:
 
 Google Analytics will only appear in production, i.e., `JEKYLL_ENV=production`
 
+### Link to specific files in header
```

Comment:
Seems like there was a consensus on **Navigation links**  ;-)

Keep it minima!™